static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,192 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
+files may be updated manually as they evolve. Other projects can then share
+these artifacts by downloading (`dvc get`) or importing (`dvc import`) them for
+use in different data processes – and these don't even have to be _DVC
+projects_, as `dvc get` works anywhere in your system.
+
+The advantages of using a data registry are:
+
+- Tracked files can be safely stored in a **centralized** remote location, with
+  the ability to create any amount of distributed copies on other remote
+  storage.
+- Several projects can **share** the same files, trusting that everyone is using
+  the same data versions.
+- Projects that import data from the registry don't need to push these large
+  files to their own [remotes](/doc/command-reference/remote), **saving space**
+  on storage – they may not even need to a remote at all, using only the local
+  <abbr>cache</abbr>.
+- Easier to manage **access control** per remote storage configured in a single
+  data registry project.
```

Comment:
related: versioning - access previous version similar easily if needed, immutability - no easy way to rewrite the existing data or model, so the is guarantee that I will have access and it does not depend on someone else will, mistakes, etc.

Reply:
> versioning - access previous version similar easily if needed

Is this a specific benefit of data registries? I think a regular project remote provides this benefit too. But maybe I'm not understanding your idea, because this doesn't seem to relate to access control to me.

> immutability - no easy way to rewrite the existing data or model...

Added note: "A possible setup would use a read-only remote, so other projects can't affect each other accidentally."